pkg/ddl/backfilling_dist_executor.go

```diff
@@ -156,6 +151,20 @@ func (s *backfillDistExecutor) Init(ctx context.Context) error {
 	return nil
 }
 
+func decodeIndexUniqueness(job *model.Job) (bool, error) {
+	unique := make([]bool, 1)
+	err := job.DecodeArgs(&unique[0])
+	if err != nil {
+		err = job.DecodeArgs(&unique)
```

Comment:
whey will it be a slice, didn't found it

Reply:
would be better to have an explicit `unique` field, ok for now

Reply:
https://github.com/pingcap/tidb/blob/7bfb7a3a99787fc33154da61e3f02551c9c7b94f/pkg/ddl/multi_schema_change.go#L388